src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -52,7 +56,7 @@
     const toggleDisabledStateOnApplyBtn = () => {
         const contentTypeOption = contentTypeSelect.querySelector('option');
         const isContentTypeSelected = contentTypeOption.innerHTML !== contentTypeOption.dataset.default;
-        const isSectionSelected = !!sectionSelect.value;
+        const isSectionSelected = sectionSelect ? !!sectionSelect.value : false;
```

Comment:
Nitpick: you can do it like this: `const isSectionSelected = !!(sectionSelect && !!sectionSelect.value);`